src/HDInsight/HDInsight/Az.HDInsight.psd1

```diff
@@ -133,7 +132,25 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Fixed miscellaneous typos across module'
+        ReleaseNotes = '* Changed Microsoft.Azure.Management.HDInsight from 2.1.0 to 5.1.0
+* Removed five cmdlets:
```

Comment:
These changes look breaking ones.

Reply:
@cormacpayne

Reply:
Yes, this is a by-design breaking change, released as a preview